includes/class-sensei-analysis.php

```diff
@@ -905,15 +905,22 @@ public function report_set_headers( $filename = '' ) {
 	 * @return object                 class instance object
 	 */
 	public function load_report_object( $name = '', $data = 0, $optional_data = null ) {
-		$object_name = 'Sensei_Analysis_' . $name . '_List_Table';
-		if ( is_null( $optional_data ) ) {
-			$sensei_analysis_report_object = new $object_name( $data );
+		if ( 'Overview' === $name ) {
+			$factory                = new Sensei_Reports_Overview_List_Table_Factory();
+			$sensei_analysis_object = $factory->create( $data );
 		} else {
-			$sensei_analysis_report_object = new $object_name( $data, $optional_data );
+			$object_name = 'Sensei_Analysis_' . $name . '_List_Table';
+			if ( is_null( $optional_data ) ) {
+				$sensei_analysis_object = new $object_name( $data );
+			} else {
+				$sensei_analysis_object = new $object_name( $data, $optional_data );
+			}
 		}
```

Comment:
This part looks the same as in `load_data_object`. Maybe extract it to a separate method to avoid duplication?

Reply:
Ooh! I paid not enough attention to the method name.

Reply:
It looks like the `load_report_object` method is not calling `prepare_items` so they are a bit different.

Reply:
So I returned the code for load_report_object: https://github.com/Automattic/sensei/pull/4973/commits/426e1ab936ce8d9e213a17344cf78dbb82e719fe